src/js/main.js

```diff
@@ -296,7 +297,47 @@ Site.Modules = {
     _this.$videos.on('mouseleave', function(event) {
       this.pause();
     });
-  }
+  },
+
+  bindScroll: function() {
+    var _this = this;
+
+    // On window scroll
+    $(window).scroll(function(event) {
+      // Get the scroll position
+      var scrollTop = $(window).scrollTop();
+
+      // ...handle it
+      _this.handleScroll(scrollTop);
+    });
+  },
+
+  handleScroll: function(scrollTop) {
+    var _this = this;
+
+    if(!_this.played && scrollTop > _this.threshold) { // First video is to play and postition is below threshold
+      _this.playAnimation();
+    }
+  },
+
+  playAnimation: function() {
+    var _this = this;
+
+    _this.played = true;
+
+    _this.$videos.each( function(index) {
+      var video = this;
+      setTimeout( function(callback) {
+        video.play();
+        setTimeout( function() {
+          video.pause();
```

Comment:
I don't think this works. It just feels broken rather than like a preview. Especially as there is no clear sign that hover is the play state. I think this should play all the way thru once and then reset. That helps solve the mobile issue too.

Reply:
When you say "I don't think this works" you mean the code or that the animation doesn't communicate what we are trying to say?
You are right, playing the whole video might look better.

Reply:
Sorry yes I dont think this is good UX. I think it should play all the way thru